dashboard/app/views/levels/editors/fields/_blockly_variables.html.haml

```diff
@@ -0,0 +1,8 @@
+%h1.control-legend{data: {toggle: "collapse", target: "#variables"}}
+  Blockly Variables
+
+#variables.in.collapse
+  = f.label :blockly_variables,'Pre-initialized Blockly Variables'
+  %p Enter a comma-separated list of variables that will be pre-populated in the variable dropdown blocks.
+  %p NOTE: Only works with Google Blockly.
```

Comment:
Do we want to make this bigger/redder/more obvious for curriculum writers? It seems like a big caveat for using this and so it might be worth having it be more attention grabbing.

Reply:
New UI:
![image](https://user-images.githubusercontent.com/8787187/126405586-9f9725d6-1f70-442a-905b-50b54e8af5d2.png)